src/components/Card/Card.js

```diff
@@ -28,10 +33,10 @@ export function Card({ image, name, temperament, className, mode, id }) {
 }
 
 Card.propTypes = {
-  image: PropTypes.object.isRequired,
+  image: PropTypes.object,
```

Comment:
try to describe image model better, I think you should have more or less the same shape of the image model (with properties url, alt, etc.) across you app so consider moving image model to some shared constant and use it everywhere in the app when it's possible